CollapseLauncher/XAMLs/MainApp/Pages/HomePage.xaml.cs

```diff
@@ -137,12 +144,33 @@ private void Page_Unloaded(object sender, RoutedEventArgs e)
         #endregion
 
         #region EventPanel
-        private void TryLoadEventPanelImage()
+        private async void TryLoadEventPanelImage()
         {
             if (regionNewsProp.eventPanel == null) return;
 
+            FileStream input = new FileStream(regionNewsProp.eventPanel.icon, FileMode.Open, FileAccess.Read);
+            MemoryStream output = new MemoryStream();
+            Bitmap bitmap = new Bitmap(input);
+
+            int width = (int)(bitmap.Width * m_appDPIScale);
+            int height = (int)(bitmap.Height * m_appDPIScale);
+
+            ProcessImageSettings settings = new ProcessImageSettings
+            {
+                Width = width,
+                Height = height,
+                HybridMode = HybridScaleMode.Off,
+                Interpolation = InterpolationSettings.Cubic
+            };
+            input.Position = 0;
+            await Task.Run(() => MagicImageProcessor.ProcessImage(input, output, settings));
+            output.Position = 0;
+
+            BitmapImage source = new BitmapImage();
+            source.SetSource(output.AsRandomAccessStream());
+
             ImageEventImgGrid.Visibility = !NeedShowEventIcon ? Visibility.Collapsed : Visibility.Visible;
-            ImageEventImg.Source = new BitmapImage(new Uri(regionNewsProp.eventPanel.icon));
+            ImageEventImg.Source = source;
```

Comment:
The stream should be disposed after the conversion. This might produce a disposal issue while the launcher is switching in between games (or reloading it) do an action for cleaning up the image files.

This is an example when the cached stream hasn't been disposed while the "Clear Image files" is performed
![image](https://github.com/neon-nyan/Collapse/assets/30566970/7acc4e04-b6e6-4bbb-a7e1-59ae410079a3)

I suggest to also create a cached image, the same method used in ``BackgroundManagement`` ->  ``GetResizedImageStream()`` method so the launcher would not need to resize the image once the same region is getting accessed.

Code snippets:
```CSharp
private async void TryLoadEventPanelImage()
{
    // If the region event panel property is null, then return
    if (regionNewsProp.eventPanel == null) return;

    // Get the event icon file info
    FileInfo iconFileInfo = new FileInfo(regionNewsProp.eventPanel.icon);
    if (!iconFileInfo.Exists) return; // Return if the event icon file doesn't exist

    // Get the cached filename and path
    string cachedFileHash = BytesToCRC32Simple(regionNewsProp.eventPanel.icon + iconFileInfo.Length);
    string cachedFilePath = Path.Combine(AppGameImgCachedFolder, cachedFileHash);

    // Init BitmapImage to load the image and the info for cached event icon file
    BitmapImage source = new BitmapImage();
    FileInfo cachedIconFileInfo = new FileInfo(cachedFilePath);
    
    // Determine if the cache icon exist and the file is completed (more than 1kB in size)
    bool isCacheIconExist = cachedIconFileInfo.Exists && cachedIconFileInfo.Length > 1 << 10;

    // Using the original icon file and cached icon file streams
    using (FileStream iconFileStream = iconFileInfo.OpenRead())
    using (FileStream cachedIconFileStream = isCacheIconExist ? cachedIconFileInfo.OpenRead() : cachedIconFileInfo.Create())
    {
        if (!isCacheIconExist)
        {
            // Get the icon image information and set the resized frame size
            ImageFileInfo iconImageInfo = await Task.Run(() => ImageFileInfo.Load(iconFileStream));
            int width = (int)(iconImageInfo.Frames[0].Width * m_appDPIScale);
            int height = (int)(iconImageInfo.Frames[0].Height * m_appDPIScale);
            ProcessImageSettings settings = new ProcessImageSettings
            {
                Width = width,
                Height = height,
                HybridMode = HybridScaleMode.Off,
                Interpolation = InterpolationSettings.Cubic
            };

            iconFileStream.Position = 0; // Reset the original icon stream position
            await Task.Run(() => MagicImageProcessor.ProcessImage(iconFileStream, cachedIconFileStream, settings)); // Start resizing
            cachedIconFileStream.Position = 0; // Reset the cached icon stream position
        }

        // Set the source from cached icon stream
        source.SetSource(cachedIconFileStream.AsRandomAccessStream());
    }

    // Set event icon props
    ImageEventImgGrid.Visibility = !NeedShowEventIcon ? Visibility.Collapsed : Visibility.Visible;
    ImageEventImg.Source = source;
    ImageEventImg.Tag = regionNewsProp.eventPanel.url;

    if (IsCustomBG)
    {
        ImageEventImgGrid.Margin = new Thickness(0, 0, 0, 16);
    }
}
```

However, The rest of the changes should be fine. Kudos for the work👍